lambda/olderThan5min.ts

```diff
@@ -1,7 +1,10 @@
-export const olderThan5min = (
-	timeStampFromDB: Date,
-	dateNow?: Date,
-): boolean => {
+export const olderThan5min = ({
+	timeStampFromDB,
+	dateNow,
+}: {
+	timeStampFromDB: Date
+	dateNow?: Date
```

Comment:
```suggestion
	now?: Date
```

We know from the type that it's a Date